SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -87,7 +88,7 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
         }
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(issue.Path, issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey,
+            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(issue.Path), issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey,
```

Comment:
We don't have any tests for it, and it's an old SQ version. Didn't know if I should add any or if it's obsolete.

Reply:
I had the same problem with this class recently. I don't think it's worth the cost; it's an obsolete version of SQ, it's a minor change, and testing the protobuf API is painful.